### PR TITLE
Flesh out the Responsive Web Design Stub

### PR DESCRIPTION
**Pull request by acenturyandabit**
Checklist:

- [ x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x ] My pull request targets the `master` branch of freeCodeCamp.
- [ x ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

**Comment by Twaha-Rahman** [comment]
@ojeytonwilliams Looks like the CI test failed to install the dependencies for some reason (The `Node.js CI / Test (12.x) (pull_request)` one), can you re-run it?

**Review by moT01 — DISMISSED** [review]
This page is pretty much hidden from the website, there's no way to find it by clicking around - although, you can make it there by entering [the URL](https://www.freecodecamp.org/learn/responsive-web-design/) directly. There's [some related discussion here.](https://github.com/freeCodeCamp/freeCodeCamp/issues/37850)

That being said, I am fine adding this in case we decide to make the intros more visible at some point. Plus its well written @acenturyandabit 👍

File: client/src/pages/learn/responsive-web-design/index.md [hunk 1]
```diff
@@ -4,7 +4,16 @@ superBlock: Responsive Web Design
...
 ## Introduction to Responsive Web Design
 
-This introduction is a stub
+Welcome to Responsive Web Design!
 
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/freeCodeCamp/edit/master/client/src/pages/learn/responsive-web-design/index.md).
- 
+Let's first take a look at what is in the scope of Responsive Web Design (and what is not).
+
+The Web part of Responsive Web Design means that the content you are creating is designed to be delivered over the web, to people's browsers. To do this, you will need to learn the language that browsers use to describe web pages: HTML (Hypertext Markup Language) for content, and CSS (Cascading Style Sheets) for Designing.
```
Comment (nhcarrigan) [inline client/src/pages/learn/responsive-web-design/index.md:11]:
```suggestion
The Web part of Responsive Web Design means that the content you are creating is designed to be delivered over the web to users' browsers. To do this, you will need to learn the language that browsers use to describe web pages: HTML (Hypertext Markup Language) for content, and CSS (Cascading Style Sheets) for design.
```

File: client/src/pages/learn/responsive-web-design/index.md [hunk 2]
```diff
@@ -4,7 +4,16 @@ superBlock: Responsive Web Design
...
+
+That brings us to design. Web Design focuses on how websites should look and behave to the end user. There are plenty of additional aspects to websites, including how to host a website so people can access it; how to protect your websites and store your users' data, and the ethos behind what should go on a website in the first place. These are somewhat harder to teach, but do require a good understanding of the capabilities of the web in modern contexts.
```
Comment (nhcarrigan) [inline client/src/pages/learn/responsive-web-design/index.md:13]:
```suggestion
That brings us to design. Web Design focuses on how websites should look and behave to the end user. There are plenty of additional aspects to websites, including how to host a website so people can access it, how to protect your websites and store your users' data, and the ethos behind what should go on a website in the first place. These are somewhat harder to teach, but do require a good understanding of the capabilities of the web in modern contexts. 
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Just a couple of grammar nitpicks.

File: client/src/pages/learn/responsive-web-design/index.md [hunk 3]
```diff
@@ -4,7 +4,16 @@ superBlock: Responsive Web Design
...
+
+That brings us to design. Web Design focuses on how websites should look and behave to the end user. There are plenty of additional aspects to websites, including how to host a website so people can access it; how to protect your websites and store your users' data, and the ethos behind what should go on a website in the first place. These are somewhat harder to teach, but do require a good understanding of the capabilities of the web in modern contexts. 
+
+And what of the last word - Responsive? Well, that brings us to the idea of modern contexts. In today's world, websites are expected to be just as accessible on laptops, tablets and phones. If you compare your user experience on a document-editing program on a computer versus a document-editing program on a mobile, the features on mobile are likely to be far more constrained. This applies in some way to websites as well, but developers have found interesting ways to adapt to the mobile experience, not in the least by harnessing powerful features built for flexible (a.k.a responsive) web displays such as `css-flexbox` and `css-grid`.
```
Comment (Manish-Giri) [inline client/src/pages/learn/responsive-web-design/index.md:15]: Do we really need to use the `css-` prefix with Flexbox and grid - 

> web displays such as `css-flexbox` and `css-grid`

It appears like they are literally called `css-flexbox` and `css-grid`. Maybe removing the `-` should fix it.

File: client/src/pages/learn/responsive-web-design/index.md [hunk 4]
```diff
@@ -4,7 +4,16 @@ superBlock: Responsive Web Design
...
+
+That brings us to design. Web Design focuses on how websites should look and behave to the end user. There are plenty of additional aspects to websites, including how to host a website so people can access it, how to protect your websites and store your users' data, and the ethos behind what should go on a website in the first place. These are somewhat harder to teach, but do require a good understanding of the capabilities of the web in modern contexts. 
+
+And what of the last word - Responsive? Well, that brings us to the idea of modern contexts. In today's world, websites are expected to be just as accessible on laptops, tablets and phones. If you compare your user experience on a document-editing program on a computer versus a document-editing program on a mobile, the features on mobile are likely to be far more constrained. This applies in some way to websites as well, but developers have found interesting ways to adapt to the mobile experience, not in the least by harnessing powerful features built for flexible (a.k.a responsive) web displays such as `css-flexbox` and `css-grid`.
```
Comment (raisedadead) [inline client/src/pages/learn/responsive-web-design/index.md:15]:
```suggestion
And what of the last word - Responsive? Well, that brings us to the idea of modern contexts. In today's world, websites are expected to be just as accessible on laptops, tablets and phones. If you compare your user experience on a document-editing program on a computer versus a document-editing program on a mobile, the features on mobile are likely to be far more constrained. This applies in some way to websites as well, but developers have found interesting ways to adapt to the mobile experience, not in the least by harnessing powerful features built for flexible (a.k.a responsive) web displays such as CSS `flexbox` and CSS `grid`.
```

@Manish-Giri 

Expanding on what your review comment means in terms of changes? If this is OK should I commit this?

Comment (Manish-Giri) [reply client/src/pages/learn/responsive-web-design/index.md:15]: @raisedadead Yes, looks good! Please commit.

File: client/src/pages/learn/responsive-web-design/index.md [hunk 5]
```diff
@@ -4,7 +4,16 @@ superBlock: Responsive Web Design
...
+
+That brings us to design. Web Design focuses on how websites should look and behave to the end user. There are plenty of additional aspects to websites, including how to host a website so people can access it, how to protect your websites and store your users' data, and the ethos behind what should go on a website in the first place. These are somewhat harder to teach, but do require a good understanding of the capabilities of the web in modern contexts. 
+
+And what of the last word - Responsive? Well, that brings us to the idea of modern contexts. In today's world, websites are expected to be just as accessible on laptops, tablets and phones. If you compare your user experience on a document-editing program on a computer versus a document-editing program on a mobile, the features on mobile are likely to be far more constrained. This applies in some way to websites as well, but developers have found interesting ways to adapt to the mobile experience, not in the least by harnessing powerful features built for flexible (a.k.a responsive) web displays such as CSS `flexbox` and CSS `grid`.
```
Comment (RandellDawson) [inline client/src/pages/learn/responsive-web-design/index.md:15]: > And what of the last word - Responsive? 

Not sure what is meant here by "last word".  That would actually be the first word in the phrase "Responsive Web Design".  Also, let's avoid using hyphen between "word" and "Responsive"?  I think this sentence would read better as:

> And what about the word Responsive?

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my comment and suggestion about the first sentence of the last paragraph.

**Review by Manish-Giri — DISMISSED** [review]
Looks good, w.r.t my [recommended change](https://github.com/freeCodeCamp/freeCodeCamp/pull/39663#pullrequestreview-498744962).

File: client/src/pages/learn/responsive-web-design/index.md (hunk 5 above)
Comment (RandellDawson) [reply client/src/pages/learn/responsive-web-design/index.md:15]: @acenturyandabit  Can you respond to my request?

Comment (acenturyandabit) [reply client/src/pages/learn/responsive-web-design/index.md:15]: Last as in "the one we haven't looked at yet in the preceding paragraphs". But it's a minor change, I'm sure our readers won't be thrown too much by it. Happy to go with your suggestion

Comment (RandellDawson) [reply client/src/pages/learn/responsive-web-design/index.md:15]: I would feel better about approving the PR with the verbiage I suggested above.

Comment (acenturyandabit) [reply client/src/pages/learn/responsive-web-design/index.md]: Alrighty, done!

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@nhcarrigan If you are ok with the changes, then feel free to approve and merge.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 Congratulations on your first pull request!

Thank you for your contribution to the page!
We are happy to accept these changes and look forward to future contributions.

**Pull request merged**